spec/features/publisher_information/visit_publisher_information_pages_spec.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+RSpec.feature "User is on the home page and can visit the publisher information pages" do
+  scenario do
+    given_im_on_the_home_page
+    when_i_click_on_the_publisher_updates_link_in_footer
+    then_i_can_see_publisher_updates_page
+    when_i_click_on_the_beta_capabilities_link_in_footer
+    then_i_can_see_beta_capabilities_page
+    when_i_click_on_the_how_to_use_publisher_link_in_footer
+    then_i_can_see_how_to_use_publisher_page
+  end
+
+  def given_im_on_the_home_page
+    visit root_path
+  end
+
+  def when_i_click_on_the_publisher_updates_link_in_footer
+    within(".govuk-footer") do
+      click_on I18n.t("publisher_information.publisher_updates.title")
```

Comment:
We don't translate links, as some people found they make the tests too confusing to read - although arguably makes the app confusing to develop on as well! But it's a convention we've stuck to thus far - see https://github.com/alphagov/content-publisher/blob/master/docs/adr/0004-editing-microcopy.md#decisionhttps://github.com/alphagov/content-publisher/blob/master/docs/adr/0004-editing-microcopy.md#decision.

Reply:
DONE. OK that's interesting.  The names of the links changed during the course of the story but I guess once it's done the likelihood of them needing to be altered is low.